src/containers/CourseOverview/CourseOverview.scss

```diff
@@ -26,8 +26,14 @@
     margin-left: 20%;
   }
 
+  &__image {
+    width: 400px;
+    height: 242px;
+    margin: 0 auto 3.5rem auto;
```

Comment:
The automatic margin here doesn't centre the image correctly, it seems to be aligned to the left which is especially clear on larger screen sizes